gateways/istio-ingress/values.yaml

```diff
@@ -56,6 +56,7 @@ gateways:
     autoscaleEnabled: true
     autoscaleMin: 1
     autoscaleMax: 5
+    # daemonsetEnabled: false #change to true to deploy as "kind: DaemonSet" otherwise "kind: Deployment"
```

Comment:
Comments are usually above the option. Maybe few more words on why someone would do this, or some wiki page link with more details ( requires VMs to have public IPs, DNS or other LB setup, advise on request/limit for CPU).

Reply:
@costinm Thank you for your review.
I will try to consider comments and docs.